### PR TITLE
Adds BMW sponsoredContent to all events in city view

### PR DESCRIPTION
**Pull request by kierangillen**
- Adds BMW sponsoredContent to all events in city view

### CODE REVIEW

File: src/lib/Scenes/City/Components/BMWEventSection/index.tsx [hunk 1]
```diff
@@ -1,57 +1,67 @@
...
+        <Box my={2} px={2}>
+          <Serif size="8">{this.props.title}</Serif>
+        </Box>
+        <Box mb={2} px={2}>
+          <Sans weight="medium" size="3">
+            Presented by BMW
+          </Sans>
+          <Box mt={1}>
+            <Serif size="3" color={color("black60")}>
+              {introText}
```
Comment (anandaroop) [inline src/lib/Scenes/City/Components/BMWEventSection/index.tsx]: Yep, so implicit in the way this is all set up now is that these sponsored content text blurbs do not contain any markup. Judging from the comps that seems like a safe assumption for now, but wanted to call it out.

File: src/lib/Scenes/Map/GlobalMap.tsx [hunk 2]
```diff
@@ -409,6 +410,10 @@ export const GlobalMapContainer = createFragmentContainer(
     fragment GlobalMap_viewer on Viewer @argumentDefinitions(citySlug: { type: "String!" }, maxInt: { type: "Int!" }) {
       city(slug: $citySlug) {
         name
+        sponsoredContent {
+          introText
+          artGuideUrl
+        }
```
Comment (anandaroop) [inline src/lib/Scenes/Map/GlobalMap.tsx]: 👍

File: src/lib/Scenes/City/Components/BMWEventSection/index.tsx (hunk 1 above)
Comment (kierangillen) [reply src/lib/Scenes/City/Components/BMWEventSection/index.tsx]: 👍

**Comment by kierangillen** [comment]
@orta I dont see rules corresponding to this style, `Serif` with `color("black60")` -- maybe need to update the component? Or should we just not use the `Markdown` component for now?

**Comment by orta** [comment]
Cool, sure, makes sense 👍

**Pull request merged**